### PR TITLE
Normal png res file

### PR DESCRIPTION
**Pull request by yschimke**
#### WHAT

Stop sample_image.png being lfs.

#### WHY

Causing issues with repo synching.

#### Checklist :clipboard:
- [ ] Add explicit visibility modifier and explicit return types for public declarations
- [ ] Run spotless check
- [ ] Run tests
- [ ] Update metalava's signature text files

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1,5 +1,5 @@
 **/test/snapshots/** filter=lfs diff=lfs merge=lfs -text
 **/figma/** filter=lfs diff=lfs merge=lfs -text
+**/docs/**/*.png filter=lfs diff=lfs merge=lfs -text
```
Comment (luizgrp) [inline .gitattributes:3]: we also want the pngs for the tests, just have to make sure that the filter of line 1 is working

**Review by luizgrp — APPROVED** [review]
LGTM

File: .gitattributes (hunk 1 above)
Comment (yschimke) [reply .gitattributes:3]: I'll test.

Comment (yschimke) [reply .gitattributes:3]: Worked

**Comment by yschimke** [comment]
Used https://stackoverflow.com/questions/35011366/move-git-lfs-tracked-files-under-regular-git

**Pull request merged**